gramps/gen/lib/tableobj.py

```diff
@@ -125,7 +125,7 @@ def get_change_display(self):
 
         """
         if self.change:
-            return str(time.strftime('%x %X', time.localtime(self.change)),
```

Comment:
I think that the following should work:

`return time.strftime('%x %X', time.localtime(self.change))`